outplacement_order_interpreter/models/outplacement.py

```diff
@@ -5,8 +5,10 @@ class Outplacement(models.Model):
     _inherit = "outplacement"
 
     interpreter_language = fields.Many2one(
-        comodel_name="res.interpreter.language",
         related="partner_id.interpreter_language",
         readonly=False)
-    interpreter_gender_preference = fields.Selection(
-        related="partner_id.interpreter_gender_preference", readonly=False)
+    interpreter_gender_preference = fields.Many2one(
+        related="partner_id.interpreter_gender_preference",
+        readonly=False)
+    # interpreter_type = fields.Many2one()
+    # interpreter_remote_type = fields.Many2one()
```

Comment:
Remove these commented lines or add a comment above why need these in the future.